Code/GraphMol/MolDraw2D/Qt/MolDraw2DQt.cpp

```diff
@@ -24,6 +24,8 @@ using namespace std;
 
 namespace RDKit {
 
+const char *rdkitQtVersion = RDK_QT_VERSION;
```

Comment:
We normally do these as strings, but I guess it's not necessary
```suggestion
const std::string rdkitQtVersion = RDK_QT_VERSION;
```